src/main/java/io/github/wistefan/mapping/annotations/SubscriptionType.java

```diff
@@ -0,0 +1,14 @@
+package io.github.wistefan.mapping.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to indicate that a certain method should be used for returning the subscription-type
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface SubscriptionType {
```

Comment:
What types can a subscription have? I think there is only one in NGSI-LD?

Reply:
Same here as above. I was following the example of an entity